tests/cases/compiler/namespaceModule.ts

```diff
@@ -0,0 +1,9 @@
+namespace z {}
```

Comment:
Call me crazy, but I don't actually think we need a new test for this change. 🐱 memberScope.errors.txt and importDeclWithDeclareModifier.errors.txt, for example, show exactly the improved error message that is intended.